lib/models/payment-processor-adapters/stripe.js

```diff
@@ -60,20 +66,22 @@ const stripeAdapter = {
    * and `message` properties
    */
   delete: function() {
-    return new Promise((resolve, reject) => {
-      stripe.customers.del(stripeProcessor.data.customer.id,
-          (err, confirmation) => {
-            if (err) {
-              return reject(new Error(err));
-            }
-
-            if (!confirmation.deleted) {
-              return reject(new Error('stripe couldn\'t delete the customer'));
-            }
-
-            return resolve(null);
-          });
-    });
+    throw new Error("you shouldn'n be here");
```

Comment:
Can we clean this up? e.g. `throw new Error('Not implemented')`, and the existing code will be in git history (no need to comment).